machine/dni/dni_3048up/busybox/conf/config

```diff
@@ -0,0 +1,6 @@
+CONFIG_SYS_EEPROM=y
+CONFIG_SYS_EEPROM_DEVICE_MTD=y
+CONFIG_SYS_EEPROM_MTD_DEVICE="/dev/mtd-sys-eeprom"
+CONFIG_SYS_EEPROM_OFFSET=0
+CONFIG_SYS_EEPROM_SIZE=256
```

Comment:
I find sys-eeprom max size in u-boot like that:
int max_size = min(TLV_TOTAL_LEN_MAX,
CONFIG_SYS_EEPROM_MAX_SIZE - sizeof(tlvinfo_header_t));
So, i modify busybox config 'CONFIG_SYS_EEPROM_SIZE' to 2048.

thanks